app/assets/javascripts/winners-bundle.js

```diff
@@ -1,4 +1,5 @@
-//= require d3
-//= require winners/c3
-//= require winners/underscore
-//= require_tree ./winners/
```

Comment:
it did work, but I like having all of the files here explicitly. What do you think?